parameter-sets/user-accounts/parameter-set.json

```diff
@@ -0,0 +1,65 @@
+{
+    "meta": {
+        "label": "User accounts",
```

Comment:
I'll rename it to Personal user accounts, to have the same term as in the Custom Dataset UI

Reply:
👍 done in [aaed917](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/aaed917b0a321eee724651010f752fddf7f088c4)